app/form_objects/provider_visa_form.rb

```diff
@@ -0,0 +1,25 @@
+class ProviderVisaForm
+  include ActiveModel::Model
+
+  attr_reader :can_sponsor_skilled_worker_visa, :can_sponsor_student_visa
```

Comment:
This might be a simpler way to cast to booleans, to avoid overriding the setter methods? (Copied from a previous project 😄 )

```suggestion
  include ActiveModel::Attributes

  attribute :can_sponsor_skilled_worker_visa, :boolean
  attribute :can_sponsor_student_visa, :boolean
```